apps/ledger-live-mobile/src/index.js

```diff
@@ -231,6 +239,11 @@ const linkingOptions = {
            * ie: "ledgerlive://wc?uri=wc:00e46b69-d0cc-4b3e-b6a2-cee442f97188@1?bridge=https%3A%2F%2Fbridge.walletconnect.org&key=91303dedf64285cbbaf9120f6e9d160a5c8aa3deb67017a3874cd272323f48ae
            */
           [ScreenName.WalletConnectDeeplinkingSelectAccount]: "wc",
+          /**
+           * @params ?platform: string
+           * ie: "ledgerlive://discover/paraswap?theme=light" will open the catalog and the paraswap dapp with a light theme as parameter
+           */
+          [ScreenName.PlatformApp]: "discover/:platform?",
```

Comment:
interesting I didn't try this exact solution, let me try that out!

Reply:
thanks for the review!

Reply:
Indeed that works! I think before having to implement the aliasing logic I had tried to keep it simple like that and it wasn't working because I had kept the "?" there.
You can check the latest changes. Thanks for this suggestion it's much better like that 👍

Reply:
Nice, glad that it works